soifunc/deband.py

```diff
@@ -21,30 +18,47 @@ def RetinexDeband(
     "medium" `threshold` in f3kdb is 48. I think that's a bit strong.
     16 might be a more sane starting point. Increase as needed.
 
-    `mask_threshold` determines how sensitive the mask is, lower values
-    should preserve more detail. It does not need to be manually scaled for bit-depth,
+    `mask_threshold` determines how sensitive the mask is.
+    Lower values should preserve more detail.
+    It does not need to be manually scaled for bit-depth,
     this function will do that automatically.
 
     This function does not add grain on its own. Use another function like
-    `kagefunc.adaptive_grain` to do that.
+    `vsdeband.sized_grain` to do that.
```

Comment:
How does this compare to `adaptive_grain`? Does it work essentially the same way?

Reply:
It's more expanded, allows for custom grainers, for the size to be adjusted, etc. The base method does not come with adaptive graining, but the `adaptive` method does. You'd call it like this:

```py
from vsdeband import sized_grain

clip = ...

grain = sized_grain.adaptive(clip, [2.0, 0.25], luma_scaling=10, **grain_settings)
```